tensorflow/python/layers/base_test.py

```diff
@@ -95,6 +96,20 @@ def testAddWeight(self):
           regularizer=regularizer)
       self.assertEqual(len(layer.losses), 1)
 
+  def testReusePartitionedVaraiblesAndRegularizers(self):
+    regularizer = lambda x: math_ops.reduce_sum(x) * 1e-3
+    partitioner = partitioned_variables.fixed_size_partitioner(3)
+    for i in xrange(2):
```

Comment:
How about `for reuse in [False, True]:`, and remove the conversion logic in 105

Reply:
+1